projects/dvl-fw/src/lib/dvl-fw.module.ts

```diff
@@ -1,12 +1,15 @@
 // refer https://angular.io/guide/styleguide#style-03-06 for import line spacing
 import { NgModule } from '@angular/core';
 
+import {MatExpansionModule} from '@angular/material/expansion';
```

Comment:
This doesn't need to be here. Expansion panel will be on make-a-vis lib, not dvl-fw

Reply:
removed from dvl-fw